### DISCUSSION TITLE
Apply min and max date

### DISCUSSION THREAD

**Discussion by pharreX19** [vazco/form-builder-documentation#20] [Q&A]
How to apply max and min date for the date field in [vazco form builder](https://www.vazco.eu/form-builder)?

**Comment by caluskrzysztof** [comment]
@pharreX19 Currently FormBuilder doesn't support max and min date for the date field. It's in our backlog to improve that, however, I can't right now provide the date of implementation. 
If you could share the goal that you want to achieve maybe we could find a walkaround for that issue. 

It's also worth mentioning that neither MUI nor Material uniforms themes support min/max date. The issue status can be found here [https://github.com/vazco/uniforms/issues/1213](url)
